resources/views/PageDesign/Partials/Header/Navigation.twig

```diff
@@ -12,7 +12,13 @@
                         <img alt="{{ companyName }}" src="{{ companyLogo }}" class="img-fluid" />
                     </a>
                 {% endif %} 
-                <button v-open-mobile-navigation class="navbar-toggler d-lg-none p-3" type="button">
+                <button v-open-mobile-navigation
+                        class="navbar-toggler d-lg-none p-3"
+                        type="button"
+                        id="mobile-navigation-toggler"
+                        for="mobile-navigation-wrapper"
+                        aria-controls="menu"
```

Comment:
Wouldn't this only work if there was an interactive element with id="menu" ?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls